atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OverflowSequenceSupplier.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.keyvalue.dbkvs.impl.oracle;
+
+
+import com.google.common.base.Supplier;
+import com.palantir.atlasdb.keyvalue.dbkvs.impl.ConnectionSupplier;
+
+public final class OverflowSequenceSupplier {
+    static final int OVERFLOW_ID_CACHE_SIZE = 1000;
```

Comment:
The variable is needed as it is used for ``INCREMENT BY OVERFLOW_ID_CACHE_SIZE`` in the ``OracleTableInitializer``.